src/logger/logger.service.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  Injectable,
+  Logger as NestLogger,
+  LoggerService,
+  Scope,
+} from '@nestjs/common';
+import {
+  Logger as WinstonLogger,
+  createLogger,
+  format,
+  transports,
+} from 'winston';
+import 'winston-daily-rotate-file';
+import { ConfigService } from '@nestjs/config';
+import { SentryService } from 'src/sentry/sentry.service';
+
+@Injectable({ scope: Scope.TRANSIENT })
+export class Logger extends NestLogger implements LoggerService {
+  public readonly logger: WinstonLogger;
+  constructor(
+    private readonly configService: ConfigService,
+    private readonly sentryService: SentryService,
+  ) {
+    super();
+    const isProduction = configService.get<string>('NODE_ENV') === 'production';
```

Comment:
This is to differentiate "dev" from deployed environments, right? So both "dev" and "volta" cache-servers would have "production" logging? Am I understanding correctly?

Reply:
Yup both of them will use production env, "notProduction" env is just for our local development environment :)